src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/JavaDependencyVerificationTests.cs

```diff
@@ -208,7 +211,8 @@ public void DependencyFulfilledByProjectReferenceExplicitMetadata ()
 				CreateAndroidLibraryTaskItem ("com.google.android.material.jar", pom.FilePath),
 			],
 			ProjectReferences = [
-				CreateAndroidLibraryTaskItem ("Google.Material.Core.csproj", null, "com.google.android:material-core", "1.0"),
+				CreateAndroidLibraryTaskItem ("Google.Material.Core.csproj", null, "com.google.android:material-core:1.0"),
+				CreateAndroidLibraryTaskItem ("Google.Material.Foo.csproj", null, "org.jetbrains.kotlin:kotlin-stdlib:2.0.0,com.google.android:material-foo:1.0"),
```

Comment:
At least one of these should use various forms of whitespace instead of a comma to separate the artifact values, e.g. newline, tab, space, etc.